Assets/Scripts/Player.cs

```diff
@@ -59,6 +63,17 @@ private void OnDisable()
 
     private void Update() 
     {
+        if (isFailed) 
+        {
+            enabled = false;
+            FindObjectOfType<GameManager>().LevelFailed();

```

Comment:
Since whatever the condition "enabled" is set to false, I think we could simplify this. Something like `enabled=isFailed || isComplete ? false : true`